CHANGES.rst

```diff
@@ -4,6 +4,8 @@
 Unreleased
 ==========
 
+- Added compatibility for PHP8, deprecated PHP7.2
```

Comment:
Looks like support for PHP 7.2 is removed and not deprecated?

Reply:
I've improved the wording with b04a777b7017.